app/models/grade.rb

```diff
@@ -0,0 +1,9 @@
+class Grade < ApplicationRecord
+  attr_accessor :number
```

Comment:
`number` is the model attribute, so you don't need to add it as attribute accessor

Reply:
Removed model attribute.